x-pack/plugins/cases/server/client/utils.test.ts

```diff
@@ -406,9 +407,9 @@ describe('utils', () => {
       `);
     });
 
-    it('creates a filter for the severity', () => {
-      expect(constructQueryOptions({ severity: CaseSeverity.CRITICAL }).filter)
-        .toMatchInlineSnapshot(`
+    it.only('creates a filter for the severity', () => {
```

Comment:
not sure if it's intended but this test has `.only`

Reply:
I am flying to Portugal tomorrow so today I committed what I was working on in case something happens to me or my laptop 😄 

But I agree, ill remove it!